README.md

```diff
@@ -69,3 +69,17 @@ tsconfig.base.json    // common typescript configuration
 tslint.json           // monorepo wide linting configuration
 yarn.lock             // the only lock file in the repo. all packages combined
 ```
+
+## Monorepo methodology
+
+Development in a monorepo environment should take place in the root of the project. However, users can consume specific packages (as opposed to consuming the entire monorepo) if they'd like. This methodology defines a structure for dependencies:
```

Comment:
"consuming the entire monorepo" is a bit confusing, isn't it?

Reply:
"as opposed to adding the entire monorepo as a dependency"?